test/cli/daemon.js

```diff
@@ -104,14 +104,13 @@ describe('daemon', () => {
       }
     })
 
-    try {
-      await daemon
-      throw new Error('Did not kill process')
-    } catch (err) {
-      expect(err.killed).to.be.true()
-
-      expect(stdout).to.include('Daemon is ready')
-    }
+    await daemon.then(
+      () => expect.fail('Did not kill process'),
+      (err) => {
+        expect(err.killed).to.be.true()
+        expect(stdout).to.include('Daemon is ready')
+      }
+    )
```

Comment:
These look equivalent to me - am I missing something?

Reply:
No, you're not missing anything - it was more an attempt at consistency with other tests that have been refactored to be more rigorous.